domain/camera/src/main/java/com/google/jetpackcamera/domain/camera/CameraXCameraUseCase.kt

```diff
@@ -212,6 +225,20 @@ constructor(
             ).build()
     }
 
+    @androidx.annotation.OptIn(ExperimentalCamera2Interop::class)
+    @OptIn(BuildCompat.PrereleaseSdkCheck::class)
+    private fun getLowLightBoostDeviceSupport() = when (BuildCompat.isAtLeastV()) {
+        true -> cameraProvider.availableCameraInfos.map { cameraInfo ->
+            Camera2CameraInfo
+                .from(cameraInfo)
+                .getCameraCharacteristic(CameraCharacteristics.CONTROL_AE_AVAILABLE_MODES)
+                ?.contains(
+                    CameraMetadata.CONTROL_AE_MODE_ON_LOW_LIGHT_BOOST_BRIGHTNESS_PRIORITY
+                )
+        }.any()
+        false -> false
+    }
+
```

Comment:
We should be checking for support on a per-lens basis. This can be rewritten as an extension property to make it easier:

```
private val CameraInfo.isLowLightBoostSupported: Boolean
    @androidx.annotation.OptIn(ExperimentalCamera2Interop::class)
    @OptIn(BuildCompat.PrereleaseSdkCheck::class)
    get() = BuildCompat.isAtLeastV() &&
        Camera2CameraInfo
            .from(this)
            .getCameraCharacteristic(CameraCharacteristics.CONTROL_AE_AVAILABLE_MODES)
            ?.contains(
                CameraMetadata.CONTROL_AE_MODE_ON_LOW_LIGHT_BOOST_BRIGHTNESS_PRIORITY
            ) ?: false
```

Then in the `CameraConstraints` (line 197), we can say:
```
lowLightBoostSupport = camInfo.isLowLightBoostSupported
```